README.md

```diff
@@ -354,5 +354,57 @@ $ curl -s http://localhost:8001/o2ims-infrastructureMonitoring/v1/alarmSubscript
 ```
 Above example will get a list of existing alarm subscriptions
 
+```
+$ curl -s -X POST --header "Content-Type: application/json" -d @subscription.json http://localhost:8000/o2ims-infrastructureMonitoring/v1/alarmSubscriptions
+```
+Above example will post an alarm subscription defined in subscription.json file 
+
 Inside _VS Code_ use the _Run and Debug_ option with the `start
 alarm-subscription-server` [configuration](.vscode/launch.json).
+<<<<<<< HEAD
+=======
+
+
+#### Alarm Notification server
+
+The alarm-notification-server should use together with alarm subscription server. The alarm subscription sever accept and manages the alarm subscriptions. The alarm notificaton servers synch the alarm subscriptions via perisist storage. To use the configmap to persist the subscriptions, the namespace "orantest" should be created at hub cluster for now (will use official oranims namespace in future). The alarm notification server accept the alerts, match the subscription filter, build and send out the alarm notification based on url in the subscription.
+
+The required Resource server URL and token can be obtained as follows:
+
+```
+$ export RESOURCE_SERVER_URL=http://localhost:8002/o2ims-infrastructureInventory/v1/
+$ export RESOURCE_SERVER_TOKEN=$(
+  oc create token -n openshift-oauth-apiserver oauth-apiserver-sa --duration=24h
```

Comment:
What is the reason to use this service account? As this is for the development environment it is probably OK, but strange. Can we instead use `oc whoami --show-token`?

Reply:
@danielerez This is related to alert to alarm mapping. Can you please answer Juan's question?

Reply:
Yeah, I guess it should work as well. But it'll require a bit more explanations

> What is the reason to use this service account? As this is for the development environment it is probably OK, but strange. Can we instead use `oc whoami --show-token`?

Yeah, I guess it should work as well. But it'll require a bit more explanation to login first, since an issue as follows could be encountered: https://access.redhat.com/solutions/6063651